app/helpers/application_helper.rb

```diff
@@ -0,0 +1,24 @@
+module ApplicationHelper
+
+  def full_name(user)
+    "#{user.first_name} #{user.last_name}"
+  end
+
+  def log_in(user)
+    session[:user_id] = user.id
```

Comment:
It's best practice to call `reset_session` immediately after successful login and while logging out.

http://stackoverflow.com/a/14949063